Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -171,7 +171,8 @@ export function runQTest(args: QTestArguments): Result {
         Cmd.flag("--qTestIgnoreQTestSkip", args.qTestIgnoreQTestSkip),
         Cmd.option("--qTestAdditionalOptions ", args.qTestAdditionalOptions, args.qTestAdditionalOptions ? true : false),
         Cmd.option("--qTestContextInfo ", qTestContextInfoPath),
-        Cmd.option("--qTestBuildType ", args.qTestBuildType || "unset")
+        Cmd.option("--qTestBuildType ", args.qTestBuildType || "unset"),
+        Cmd.option("--testSourceDir", args.testSourceDir)

```

Comment:
>args.testSourceDir [](start = 38, length = 18)

This should be path, instead of string. But then, since this is just a directory, you can do `Artifact.none(args.testSourceDir)`.

Reply:
Artifact.none does not accept RelativePath

Reply:
If relative path no need for artifact...